### PR TITLE
feat: added .printer 

### PR DESCRIPTION
**Pull request by oim1**
## Description

.printer added. Gives a link to the tooltip. 

## Test Results

(.simbrief no longer in this PR)

youtubecommands2

## Discord Username

oim#0001

### CODE REVIEW

**Comment by benw202** [comment]
.simbrief already exists as a commands alias.

It shows the simbrief airframe download as well as taking you to the docs guide on simbrief integration.

File: src/commands/printer.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import { makeEmbed } from '../lib/embed';
+
+export const printer: CommandDefinition = {
+    name: 'printer',
+    description: 'Provides a link to the FlyByWire printer tutorial video',
+    category: CommandCategory.FBW,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'FlyByWire A32NX | How do I use the printer?',
+        description: 'Watch a tutorial on how to use the FlyByWire A32NX printer [here](https://www.youtube.com/watch?v=i3ughyFtnCM)',
+        footer: { text: 'Note: Using the printer may cause a CTD. This is a random issue and we are currently looking into it'}
```
Comment (NUTTTTTTTT) [inline src/commands/printer.ts:12]:
```suggestion
        footer: { text: 'Warning: Using the printer may cause a crash to desktop. This issue occurs randomly and we are currently looking for ways to reproduce and resolve it.}
```

File: src/commands/simbrief.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import { CommandCategory } from '../constants';
+import { makeEmbed } from '../lib/embed';
+
+export const simbrief: CommandDefinition = {
+    name: 'simbrief',
+    description: 'Provides a link to the FlyByWire Simbrief integration tutorial video.',
+    category: CommandCategory.FBW,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'FlyByWire A32NX | Simbrief Integration',
+        description: 'Watch a tutorial on how to use the FlyByWire A32NX Simbrief Integration [here](https://youtu.be/l_5J7ZuEEtk)',
```
Comment (NUTTTTTTTT) [inline src/commands/simbrief.ts:11]: I suggest adding an explanation of what this actually means for less experienced users who would typically be the target audience for such command, as well as correctly capitalising the SimBrief name. For example:

```suggestion
        description: 'Please watch our video tutorial on how to use the FlyByWire A32NX SimBrief Integration [here.](https://youtu.be/l_5J7ZuEEtk) This functionality will allow you to import your SimBrief flight plan, including the route, required fuel, passengers and their luggage.',
```

Comment (NUTTTTTTTT) [inline src/commands/simbrief.ts:11]: Also suggest adding a footer to this command to establish that the SB username is now entered in the flyPad not MCDU, and explaining the W+B page is different but functions the same, this is all expected to move to the flyPad so wording in a way that pre-empts that change would be best.

File: src/commands/printer.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import { CommandCategory } from '../constants';
+import { makeEmbed } from '../lib/embed';
+
+export const printer: CommandDefinition = {
+    name: 'printer',
+    description: 'Provides a link to the FlyByWire printer tutorial video',
+    category: CommandCategory.FBW,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'FlyByWire A32NX | How do I use the printer?',
+        description: 'Watch a tutorial on how to use the FlyByWire A32NX printer [here](https://www.youtube.com/watch?v=i3ughyFtnCM)',
```
Comment (NUTTTTTTTT) [inline src/commands/printer.ts:11]:
```suggestion
        description: 'Please watch our video tutorial on how to use the FlyByWire A32NX printer [here.](https://www.youtube.com/watch?v=i3ughyFtnCM)',
```

File: src/commands/simbrief.ts (hunk 2 above)
Comment (benw202) [reply src/commands/simbrief.ts:11]: Same here. No need for another simbrief command

File: src/commands/printer.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+import { makeEmbed } from '../lib/embed';
+
+export const printer: CommandDefinition = {
+    name: 'printer',
+    description: 'Provides a link to the FlyByWire printer tutorial video',
+    category: CommandCategory.FBW,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'FlyByWire A32NX | How do I use the printer?',
+        description: 'Please watch our video tutorial on how to use the FlyByWire A32NX printer [here](https://www.youtube.com/watch?v=i3ughyFtnCM)',
+        footer: { text: 'Warning: Using the printer may cause a crash to desktop. This issue occurs randomly and we are currently looking for ways to reproduce and resolve it'}
```
Comment (NUTTTTTTTT) [inline src/commands/printer.ts:12]:
```suggestion
        footer: { text: 'Warning: Using the printer may cause a crash to desktop. This issue occurs randomly and we are currently looking for ways to reproduce and resolve it.'}
```

File: src/commands/printer.ts [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+import { CommandCategory } from '../constants';
+import { makeEmbed } from '../lib/embed';
+
+export const printer: CommandDefinition = {
+    name: 'printer',
+    description: 'Provides a link to the FlyByWire printer tutorial video',
+    category: CommandCategory.FBW,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'FlyByWire A32NX | How do I use the printer?',
+        description: 'Please watch our video tutorial on how to use the FlyByWire A32NX printer [here](https://www.youtube.com/watch?v=i3ughyFtnCM)',
```
Comment (NUTTTTTTTT) [inline src/commands/printer.ts:11]:
```suggestion
        description: 'Please watch our video tutorial on how to use the FlyByWire A32NX printer [here.](https://www.youtube.com/watch?v=i3ughyFtnCM)',
```

**Pull request merged**